src/blazingmq/_ext.pyi

```diff
@@ -44,7 +44,16 @@ class Session:
         on_message: Optional[Callable[[Message, MessageHandle], None]] = None,
         broker: bytes,
         message_compression_algorithm: CompressionAlgorithmType,
-        timeout: Optional[float] = None,
+        num_processing_threads: Optional[int] = None,
+        blob_buffer_size: Optional[int] = None,
+        channel_high_watermark: Optional[int] = None,
+        event_queue_watermarks: Optional[tuple[int, int]] = None,
+        stats_dump_interval: Optional[int | float] = None,
+        connect_timeout: Optional[int | float] = None,
```

Comment:
Passing in `class Timeouts` will make for a cleaner interface.

Reply:
Resolved in commit fe22261e202d81e27d00b7ba080ca772103e1c45.